willow-eventhandler/src/main/java/com/nitorcreations/willow/eventhandler/SendMailEventHandler.java

```diff
@@ -0,0 +1,143 @@
+package com.nitorcreations.willow.eventhandler;
+
+import java.util.Properties;
+import java.util.logging.Logger;
+
+import javax.mail.Authenticator;
+import javax.mail.Message;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+import com.nitorcreations.willow.messages.event.EventMessage;
+
+/**
+ * Event handler to send email with SMTP.
+ * 
+ * @author mtommila
+ */
+public class SendMailEventHandler implements EventHandler {
+
+  private Logger logger = Logger.getLogger(getClass().getName());
+
+  private String host;
+  private int port = 25;
+  private String username;
+  private String password;
+  private boolean ssl;
+  private boolean tls;
+  private String from;
```

Comment:
Make the to and from fields of type InternetAddress and handle conversion already in setters -> better get the feedback of wrong config in startup than only after the first message is sent.